go.graphics.swing/src/main/java/go/graphics/swing/text/JOGLTextDrawer.java

```diff
@@ -19,32 +19,43 @@
 import go.graphics.text.EFontSize;
 import go.graphics.text.TextDrawer;
 
-import java.awt.Font;
+import java.awt.*;
```

Comment:
Please configure the auto formatter as described in https://github.com/jsettlers/settlers-remake/wiki/Developer's-Guide#development-guidelines